kie-server-parent/kie-server-api/pom.xml

```diff
@@ -38,11 +38,17 @@
   </properties>
 
   <dependencies>
+    <dependency>
+        <groupId>org.projectlombok</groupId>
+        <artifactId>lombok</artifactId>
+        <version>1.18.32</version>
+        <scope>provided</scope>
+    </dependency>
```

Comment:
I don't think we should / need to add lombok here, we could just remove all the related annotations from the dummy classes

Reply:
removed